index.ts

```diff
@@ -0,0 +1,93 @@
+async function askForStopCode(): Promise<string> {
+    const reader = require("readline");
+    const inquirer = reader.createInterface({
+        input: process.stdin,
+        output: process.stdout
+    });
+    const questionInit = (questionText: string) =>
+        new Promise<string>(resolve => inquirer.question(questionText, resolve));
+    let code: string = await questionInit("Please enter a stop code: ");
+    console.log(code);
+    inquirer.close();
+    return code;
+}
+
+type jsonBus = {

```

Comment:
This might be better as an interface, also the first letter should be capital for interfaces